File: estore/jestore/src/org/rt/worker/Handler.java
package org.rt.worker;

import org.json.*;


public class Handler{

    public static final String get(String uri, JSONObject headers, JSONObject args, String data){
    
        return "1";
    
    }

    public static final String post(String uri, JSONObject headers, JSONObject args, String data){
    
        return "2";
    
    }

    public static final String put(String uri, JSONObject headers, JSONObject args, String data){
 
        return "3";
    
    }

    public static final String patch(String uri, JSONObject headers, JSONObject args, String data){
    
        return "4";
    
    }

    public static final String delete(String uri, JSONObject headers, JSONObject args, String data){
   
        return "5";
    
    }







}

File: estore/pyestore/handler.py
from estore import get_events_by_device


def get(session, uri, headers, args, data):

    if uri == "/api/pyestore_async/events":
        return get_events_by_device(session, args)


def post(session, uri, headers, args, data):
    return 2


def put(uri, headers, args, data):
    return 3


def patch(uri, headers, args, data):
    return 4


def delete(uri, headers, args, data):
    return 5
